leocc.hpp

```diff
@@ -271,6 +271,15 @@ class NodeBlockStmt: public NodeStmt {
     void codegen() override;
 };
 
+class NodeFunctionDef : public Node {
+public:
+    string functionName;
+    NodeBlockStmt* body;//the literal body of the function
```

Comment:
where are the fields for declspec and declarator? dont you need those?
`func-def = declspec declarator "{" block-stmt`

Reply:
declspec might not be necessary since we only have int for now, but you definitely need declarator so you know how many levels of pointer you have.